auraed/src/cells/mod.rs

```diff
@@ -0,0 +1,24 @@
+pub(crate) use cell::Cell;
```

Comment:
i'm finding it weird that "cells" is a crate outside "runtime" as it only exists as a concept inside runtime.

does it need to be a separate crate?

Reply:
We can nest it under runtime. Or just drag the files to runtime. One benefit of small crates to keep in mind is build caching is done per crate.

Reply:
I think we should keep it nearer the cell service. it provides clearer visibility expectations.